egs2/wsj0_2mix_spatialized/enh1/local/convert2wav.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
```

Comment:
looks similar to `egs2/wsj0-2mix/enh1/local/convert2wav.sh`. If it is identical, we may just use the symlink.

Reply:
Done.